code/crates/core-consensus/src/handle/propose.rs

```diff
@@ -90,7 +90,7 @@ where
     Ctx: Context,
 {
     if state.height() != local_value.height {
-        warn!(
+        debug!(
```

Comment:
Is this supposed to happen? We receive a proposed value at the wrong height/round?